From db3c3d82184ef2819c3317e1f8d55a652966783b Mon Sep 17 00:00:00 2001
From: Michal Schmidt <mschmidt@redhat.com>
Date: Wed, 13 Jun 2012 00:00:52 +0200
Subject: [PATCH] Revert "F17: units: do not use Type=idle yet"

This reverts commit 0b4dd49539db110d27e327a5a34b739d634f7341.

Now that Type=idle bugs have been fixed and it uses acceptable timeouts,
we can enable it in F17.
---
 units/console-getty.service.m4.in |    1 +
 units/console-shell.service.m4.in |    1 +
 units/emergency.service.in        |    1 +
 units/getty@.service.m4           |    1 +
 units/rescue.service.m4.in        |    1 +
 units/serial-getty@.service.m4    |    1 +
 6 files changed, 6 insertions(+)

diff --git a/units/console-getty.service.m4.in b/units/console-getty.service.m4.in
index cc8c364..9d80d4b 100644
--- a/units/console-getty.service.m4.in
+++ b/units/console-getty.service.m4.in
@@ -31,6 +31,7 @@ Before=getty.target
 
 [Service]
 ExecStart=-/sbin/agetty --noclear -s console 115200,38400,9600
+Type=idle
 Restart=always
 RestartSec=0
 UtmpIdentifier=cons
diff --git a/units/console-shell.service.m4.in b/units/console-shell.service.m4.in
index b0ced10..90c7c78 100644
--- a/units/console-shell.service.m4.in
+++ b/units/console-shell.service.m4.in
@@ -33,6 +33,7 @@ Environment=HOME=/root
 WorkingDirectory=/root
 ExecStart=-/sbin/sulogin
 ExecStopPost=-@SYSTEMCTL@ poweroff
+Type=idle
 StandardInput=tty-force
 StandardOutput=inherit
 StandardError=inherit
diff --git a/units/emergency.service.in b/units/emergency.service.in
index a1fc2ea..c76a738 100644
--- a/units/emergency.service.in
+++ b/units/emergency.service.in
@@ -18,6 +18,7 @@ ExecStartPre=-/bin/plymouth quit
 ExecStartPre=-/bin/echo 'Welcome to emergency mode. Use "systemctl default" or ^D to enter default mode.'
 ExecStart=-/sbin/sulogin
 ExecStopPost=@SYSTEMCTL@ --fail --no-block default
+Type=idle
 StandardInput=tty-force
 StandardOutput=inherit
 StandardError=inherit
diff --git a/units/getty@.service.m4 b/units/getty@.service.m4
index f57ba99..7022743 100644
--- a/units/getty@.service.m4
+++ b/units/getty@.service.m4
@@ -42,6 +42,7 @@ ConditionPathExists=/dev/tty0
 [Service]
 Environment=TERM=linux
 ExecStart=-/sbin/agetty %I 38400
+Type=idle
 Restart=always
 RestartSec=0
 UtmpIdentifier=%I
diff --git a/units/rescue.service.m4.in b/units/rescue.service.m4.in
index 76f8946..6b601b9 100644
--- a/units/rescue.service.m4.in
+++ b/units/rescue.service.m4.in
@@ -31,6 +31,7 @@ m4_ifdef(`TARGET_MEEGO',
 ExecStart=-/bin/bash -c "exec ${SINGLE}"',
 `ExecStart=-/sbin/sulogin'))))
 ExecStopPost=-@SYSTEMCTL@ --fail --no-block default
+Type=idle
 StandardInput=tty-force
 StandardOutput=inherit
 StandardError=inherit
diff --git a/units/serial-getty@.service.m4 b/units/serial-getty@.service.m4
index 66dfeed..6f6a99f 100644
--- a/units/serial-getty@.service.m4
+++ b/units/serial-getty@.service.m4
@@ -38,6 +38,7 @@ IgnoreOnIsolate=yes
 [Service]
 Environment=TERM=vt102
 ExecStart=-/sbin/agetty -s %I 115200,38400,9600
+Type=idle
 Restart=always
 RestartSec=0
 UtmpIdentifier=%I
